python/cudf/cudf/core/dataframe.py

```diff
@@ -127,6 +123,20 @@ class DataFrame(object):
     3    3  2018-10-07T12:00:03.000
     4    4  2018-10-07T12:00:04.000
 
+    Build DataFrame via list of rows as tuples:
+
+    >>> import cudf
+    >>> df = cudf.DataFrame([
+    >>>   (5, 'cats', 'jump', 7.5),                # 3 columns
+    >>>   (2, 'dogs', 'dig', np.nan),              # 3 columns
+    >>>   (3, 'cows', 'moo', -2.1, 'occasionally') # 4 columns
+    >>> ])
+    >>> df
+       0     1     2     3             4
+    0  5  cats  jump  null          None
```

Comment:
This doesn't make sense, this should either be `NaN` for 3 and `null` for 4 or both should be `null`. Either we convert the `NaN` to a `null` or we leave it as a `NaN`, but `None` is invalid here

Reply:
`cudf.Series([None, None, 'occasionally'])` interprets the dtype to be `object`, thereby making `None` a valid value. I think the same thing is happening here. I think that should be a separate pr.

However, the input/output are mismatched in the docs. I'm fixing it now.

Reply:
```
>>> s = Series([None])
>>> s
0    null
dtype: float64
>>> s = Series([None], dtype='object')
>>> s
0    None
dtype: object
>>> s.null_count
1
```

Reply:
👍